tools/tensorflow_docs/api_generator/parser_test.py

```diff
@@ -610,7 +610,7 @@ def test_downgrade_h1_docstrings(self):
     downgrader = parser._DowngradeH1Keywords()
     doc = downgrader(h1_docstring)
     self.assertIn('\n  ```\n  # comment\n  ```', doc)
-    self.assertIn('\nArguments:', doc)
+    self.assertIn('\nArgs:', doc)
```

Comment:
This is correct (I think), but you'll likely need to change code elsewhere to guarantee that `Arguments` is not generated. For example, I updated this function call: https://github.com/tensorflow/tensorflow/pull/45420/files#diff-23d5b1c21f17ffbee338aed996ac92e9e247f940a461464e1810bc67801e6769